packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -389,10 +629,14 @@ contract Colosseum is Initializable, Semver {
             return ChallengeStatus.NONE;
         }
 
+        if (_challenge.outputRoot != bytes32(0)) {
```

Comment:
Do you intend to add `outputRoot` to check the status? If so, you don't need to add this. You can check whether `_challenge.next` equals to `address(0)`